calico_containers/calico_ctl/node.py

```diff
@@ -510,3 +528,28 @@ def install_kubernetes(kubernetes_plugin_dir):
         except OSError:
             print "ERROR: Unable to set permissions on Kubernetes plugin"
             sys.exit(1)
+
+
+def install_rkt(rkt_plugin_dir):
```

Comment:
This is the exact same function as `install_kubernetes`, think you can commonize it in some way?